.github/workflows/coverage_converter_publish_actions.yml

```diff
@@ -0,0 +1,134 @@
+name: Coverage Converter Publish Actions
+
+on:
+  push:
+    paths:
+      - 'metrics/coverage_converter/**'
+    branches:
+      - 'master'
+
+jobs:
+  macos-actions:
+    name: Coverage Converter MacOs Build Actions
+    runs-on: macOS-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          ref: ${{ github.ref }}
+
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: beta
+          version: 2.10.0-110.5.beta
+
+      - name: Get Dependencies
+        run: pub get
+        working-directory: metrics/coverage_converter/
+
+      - name: Build Project
+        working-directory: metrics/coverage_converter/
+        run: dart2native bin/main.dart -o ./coverage_converter_macos
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: coverage_converter_macos
+          path: metrics/coverage_converter/coverage_converter_macos
+          if-no-files-found: error
+
+  linux-actions:
+    name: Coverage Converter Linux Build Actions
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          ref: ${{ github.ref }}
+
+      - uses: cedx/setup-dart@v2
+        with:
+          release-channel: beta
+          version: 2.10.0-110.5.beta
+
+      - name: Get dependencies
+        run: pub get
+        working-directory: metrics/coverage_converter/
+
+      - name: Build Project
+        working-directory: metrics/coverage_converter/
+        run: dart2native bin/main.dart -o ./coverage_converter_linux
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: coverage_converter_linux
+          path: metrics/coverage_converter/coverage_converter_linux
+          if-no-files-found: error
+
+
+  release-actions:
```

Comment:
Should this job depend on the previous one to avoid publishing the app when any of the builds failed? Also, should the `linux-actions` depend on `macos-actions` to stop the workflow once tests/macOS build failed?